Grbl_Esp32/src/Settings.h

```diff
@@ -454,30 +458,32 @@ class WebCommand : public Command {
                const char*   grblName,
                const char*   name,
                Error (*action)(char*, WebUI::AuthenticationLevel)) :
-        Command(description, type, permissions, grblName, name),
+    // At some point we might want to be more subtle, but for now we block
+    // all web commands in Cycle and Hold states, to avoid crashing a
+    // running job.
+    Command(description, type, permissions, grblName, name, notCycleOrHold),
```

Comment:
This line is wrong.  I made a git mistake and lost an important change.  A fix is forthcoming.